### PR TITLE
Issue #SB-18938 fix: background notification navigation added, course…

### PR DESCRIPTION
**Pull request by raghav14**
… loaders removed

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -351,6 +352,10 @@ export class AppComponent implements OnInit, AfterViewInit {
         this.events.publish('notification-status:update', { isUnreadNotifications: true });
       });
       this.notificationSrc.setNotificationDetails(data);
+      console.log(this.isForground);
```
Comment (AmiableAnil) [inline src/app/app.component.ts:355]: Remove the log

File: src/app/app.component.ts [hunk 2]
```diff
@@ -412,10 +417,15 @@ export class AppComponent implements OnInit, AfterViewInit {
       this.telemetryGeneratorService.generateInterruptTelemetry('resume', '');
       this.splashScreenService.handleSunbirdSplashScreenActions();
       this.checkForCodeUpdates();
+      this.notificationSrc.handleNotification();
+      this.isForground = true;
+      console.log(this.isForground);
```
Comment (AmiableAnil) [inline src/app/app.component.ts:422]: Remove log

File: src/app/app.component.ts [hunk 3]
```diff
@@ -412,10 +417,15 @@ export class AppComponent implements OnInit, AfterViewInit {
...
     });
 
     this.platform.pause.subscribe(() => {
       this.telemetryGeneratorService.generateInterruptTelemetry('background', '');
+      this.isForground = false;
+      console.log(this.isForground);
```
Comment (AmiableAnil) [inline src/app/app.component.ts:428]: Remove log

File: src/app/app.component.ts [hunk 4]
```diff
@@ -412,10 +417,15 @@ export class AppComponent implements OnInit, AfterViewInit {
       this.telemetryGeneratorService.generateInterruptTelemetry('resume', '');
       this.splashScreenService.handleSunbirdSplashScreenActions();
       this.checkForCodeUpdates();
+      this.notificationSrc.handleNotification();
+      this.isForground = true;
```
Comment (swayangjit) [inline src/app/app.component.ts:421]: isForground  should be isForeground

**Pull request merged**